extensions/markdown-language-features/src/features/previewManager.ts

```diff
@@ -106,7 +106,10 @@ export class MarkdownPreviewManager extends Disposable implements vscode.Webview
 			preview = this.createNewDynamicPreview(resource, settings);
 		}
 
-		preview.update(resource);
+		preview.update(
```

Comment:
Calling update here will force the preview to be re-rendered right away. Can the fragment be passed to the two calls above instead?

Reply:
Isn't `preview.update` already being called here? The PR just adds a second argument. Or are you saying including the second argument forces a re-render which wouldn't otherwise happen?

I didn't quite follow "the two calls above", could you clarify which lines you're referring to?